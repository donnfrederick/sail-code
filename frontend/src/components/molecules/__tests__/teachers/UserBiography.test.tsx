import StudentBiography from 'components/molecules/teachers/UserBiography'
import * as React from 'react'
import * as renderer from 'react-test-renderer'

test('render <StudentBiography />', () => {
  const tree = renderer
    .create(
      <StudentBiography country={'Japan'} hobbies={[]} sex={1} purposes={[]} />
    )
    .toJSON()
  expect(tree).toMatchSnapshot()
})
