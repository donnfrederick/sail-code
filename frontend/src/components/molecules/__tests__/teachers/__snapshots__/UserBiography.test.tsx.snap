// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render <StudentBiography /> 1`] = `
<div
  className="sc-bdVaJa faUWMX"
>
  <div
    className="sc-bwzfXH dgSeGn"
  >
    <div
      className="sc-htpNat iJYhYB"
    >
      性別
    </div>
    <div
      className="sc-bxivhb bUZGRr"
    >
      男性
    </div>
  </div>
  <div
    className="sc-bwzfXH dgSeGn"
  >
    <div
      className="sc-htpNat iJYhYB"
    >
      国籍
    </div>
    <div
      className="sc-bxivhb bUZGRr"
    >
      Japan
    </div>
  </div>
  <div
    className="sc-bwzfXH dgSeGn"
  >
    <div
      className="sc-htpNat iJYhYB"
    >
      興味
    </div>
    <div
      className="sc-bxivhb bUZGRr"
    >
      
    </div>
  </div>
</div>
`;
